openwisp_controller/geo/migrations/0002_default_groups_permissions.py

```diff
@@ -4,7 +4,7 @@
 
 
 class Migration(migrations.Migration):
-    dependencies = [('openwisp_users', '0004_default_groups'), ('geo', '0001_initial')]
+    dependencies = [('geo', '0001_initial')]

```

Comment:
I guess the safe road here is to specify the swappable dependency to users as in the other subapps, don't you think so?